### ISSUE TITLE
changes in NONMEM repo

### ISSUE THREAD

**Issue by RichardHooijmaijers** [billdenney/Pharmacometrics-Docker#2]
Hi @billdenney 

It seems that something is changed in the NONMEM repository. Downloading the NONMEM zip does not work as expected anymore, e.g. something like this:
`wget -nv --no-check-certificate "https://nonmem.iconplc.com/nonmem741/NONMEM7.4.1.zip"`  
will now issue an error "Username/Password Authentication Failed.". I tried various options like --ask-password or --password but without success. Did you experience problems and have a possible solution?

**Comment by billdenney** [comment]
Icon appears to have implemented a new web server which yields an HTTP 401 error from wget but works from a web browser.  A workaround is to download it manually and serve the file from a local location, and I'll ask Bob if he knows what has changed and a way to work around it.

**Comment by billdenney** [comment]
The updates to the website make the `--auth-no-challenge` flag for `wget` is required.  I'll add it now, and hopefully it'll work.

**Comment by billdenney** [comment]
I just fixed it in 5037a5c

**Comment by RichardHooijmaijers** [comment]
Hi @billdenney Thank you for the quick action! I just tried it and it seemed to work, but only for a NONMEM version I downloaded before. For instance if I try it with version 7.4.2 I get "Username/Password Authentication Failed." while for a version I downloaded manually before it works? For me it's no problem to manually download the zip file and add it to the installation directory, not sure if this would work for everybody.
